README.md

```diff
@@ -1,19 +1,26 @@
-<p align="center">
-<img src="https://github.com/jina-ai/jina/blob/master/.github/logo-only.gif?raw=true" alt="Jina banner" width="200px">
-</p>
+# ImagePaddlehubEncoder
 
-# Image Normalizer
+**ImagePaddlehubEncoder** encodes `Document` content from a ndarray, potentially B x (Channel x Height x Width) into a ndarray of `B x D`. Internally, **ImagePaddlehubEncoder** wraps the models from [paddlehub](https://github.com/PaddlePaddle/PaddleHub)
 
-### Description
-The image normalizer resizes, crops and normalizes images.
 
-### Parameters
-The following parameters can be used:
+## Example:
 
-- `resize_dim` (int): The size of the image after resizing
-- `target_size` (tuple or int): The dimensions to crop the image to (center cropping is used)
-- `img_mean` (tuple, default (0,0,0)): The mean for normalization
-- `img_std` (tuple, default (1,1,1)): The standard deviation for normalization
-- `channel_axis` (int): The channel axis in the images used
-- `target_channel_axis` (int): The desired channel axis in the images. If this is not equal to the channel_axis, the axis is moved.
+Here is an example usage of the clip encoder.
 
+```python
+    def process_response(resp):
+        ...
+
+    f = Flow().add(uses={
+        'jtype': ImagePaddlehubEncoder.__name__,
+        'with': {
+            'default_batch_size': 32,
+            'model_name': 'xception71_imagenet',
+        },
+        'metas': {
+            'py_modules': ['paddle_image.py']
+        }
+    })
+    with f:
+        f.post(on='/test', inputs=(Document(blob=np.ones((224, 224, 3))) for _ in range(25)), on_done=process_response)
+```
```

Comment:
I find these two section very useful when looking at executors (just an example from https://github.com/jina-ai/executor-template): 

```
### Inputs 

`Document` with `blob` of the shape `256`.

### Returns

`Document` with `embedding` fields filled with an `ndarray` of the shape `embedding_dim` (=128, by default) with `dtype=nfloat32`.
```

Maybe we can add this here as well?

Reply:
Thanks! I will update readme according to the template